scout/server/blueprints/cases/controllers.py

```diff
@@ -363,15 +363,17 @@ def case(store, institute_obj, case_obj):
     pheno_groups = institute_obj.get("phenotype_groups") or PHENOTYPE_GROUPS
 
     # If case diagnoses are a list of integers, convert into a list of dictionaries
-    omim_terms = {}
+    disease_terms = {}
     case_diagnoses = case_obj.get("diagnosis_phenotypes", [])
     if case_diagnoses:
         if isinstance(case_diagnoses[0], int):
             case_obj = store.convert_diagnoses_format(case_obj)
         # Fetch complete OMIM diagnoses specific for this case
-        omim_terms = {
+        disease_terms = {
             term["disease_id"]: term
-            for term in store.case_omim_diagnoses(case_obj.get("diagnosis_phenotypes"))
+            for term in store.case_diseases(
+                case_disease_list=case_obj.get("diagnosis_phenotypes"), filter_project=None
```

Comment:
The fix is passing a null filter_project, because the default projection in that function is {"hpo_terms": 0, "genes": 0}